src/platforms/px4_module.h

```diff
@@ -472,6 +472,12 @@ __EXPORT void PRINT_MODULE_DESCRIPTION(const char *description);
  */
 __EXPORT void PRINT_MODULE_USAGE_NAME(const char *executable_name, const char *category);
 
+/**
+ * @brief Specify a subcategory.
```

Comment:
Perhaps mention that subcategories are optional?

> @brief Specify a subcategory (optional).

Reply:
Applied